crates/iota/src/fire_drill.rs

```diff
@@ -161,20 +160,14 @@ async fn update_next_epoch_metadata(
     new_config.protocol_key_pair =
         KeyPairWithPath::new(IotaKeyPair::Ed25519(new_protocol_key_pair));
 
-    let iota_system_state = iota_client
+    let self_validator = iota_client
```

Comment:
I think this needs to remain active_validators, otherwise only committee members can do the fire_drill?

Reply:
Thanks for pointing this out, fixed at [70f9bb7](https://github.com/iotaledger/iota/pull/6015/commits/70f9bb75e2846a87d6050621865988c78a2ed806) and [4db4975](https://github.com/iotaledger/iota/pull/6015/commits/4db4975000161c937a7b5469ad04c996dc3be6cf)